### ISSUE TITLE
Font-awesome-sass is not compatible with sassc 2.0.

### ISSUE THREAD

**Issue by kevin-jj** [FortAwesome/font-awesome-sass#168]
Sassc 1.12.1 works well but sassc 2.0 does not. It looks like that Sassc 2.0.0: [Remove dependency on Ruby Sass](https://github.com/sass/sassc-ruby/pull/85)

### Exception:

```ruby
rake aborted!
LoadError: cannot load such file -- sass
/usr/local/bundle/gems/bootsnap-1.3.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
/usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:287:in `block in require'
/usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:253:in `load_dependency'
/usr/local/bundle/gems/activesupport-5.2.1/lib/active_support/dependencies.rb:287:in `require'
/usr/local/bundle/gems/font-awesome-sass-5.5.0/lib/font-awesome-sass.rb:48:in `configure_sass'
/usr/local/bundle/gems/font-awesome-sass-5.5.0/lib/font-awesome-sass.rb:13:in `load!'
/usr/local/bundle/gems/font-awesome-sass-5.5.0/lib/font-awesome-sass.rb:78:in `<main>'
/usr/local/bundle/gems/bootsnap-1.3.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `require'
/usr/local/bundle/gems/bootsnap-1.3.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `block in require_with_bootsnap_lfi'
/usr/local/bundle/gems/bootsnap-1.3.2/lib/bootsnap/load_path_cache/loaded_features_index.rb:65:in `register'
/usr/local/bundle/gems/bootsnap-1.3.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:20:in `require_with_bootsnap_lfi'
/usr/local/bundle/gems/bootsnap-1.3.2/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:29:in `require'
``` 

@supercodepoet  any help?  Thanks

**Comment by chentianwei411** [comment]
yes, I have the similar problem:
in：
An error occurred while installing sassc (2.0.0), and Bundler cannot continue.
Make sure that `gem install sassc -v '2.0.0' --source
'https://gems.ruby-china.com/'` succeeds before bundling.

**Comment by czj** [comment]
I think the fix is trivial : the code references `sass` instead of `sassc` but the dependencies list the `sassc` gem like before. Hope #169 gets merged quickly :)

**Comment by czj** [comment]
Thanks. Did you forget to merge #169 ?

**Comment by kevin-jj** [comment]
I have no authorization to get it merged. @bobmaerten @czj

**Comment by czj** [comment]
I'm sorry @kevin-jj is this a @FortAwesome/font-awesome-sass access right issue or an issue that we could fix on our own repo ?

**Comment by kevin-jj** [comment]
It had better have @supercodepoet get it merged and roll out a new release.  My current solution is to add sass gem in Gemfile.

**Comment by supercodepoet** [comment]
Should be fixed in latest 5.5.0.1 release
